src/openrct2-ui/interface/Window.cpp

```diff
@@ -749,6 +749,16 @@ void Window::Close()
     window_close(*this);
 }
 
+void Window::CloseAllWindowsExceptNumberAndClass(rct_windownumber windowNumber, WindowClass windowClass)
+{
+    window_close_all_except_number_and_class(windowNumber, windowClass);
+}
+
+void Window::CloseByClass(WindowClass windowClassification)
```

Comment:
How about (I hate naming methods) CloseByClass(), CloseAllThisClass(), or something similar that calls window_close_by_class(classification), rather than a static method?